src/css/components/_title-bar.scss

```diff
@@ -0,0 +1,41 @@
+.vjs-title-bar {
+
+  // At a base inherited font-size of 10px, the title bar overall height should
+  // be 96px with the area of text occupying the first 48px and the rest being
+  // padding. This leaves plenty of room for the gradient to fade to
+  // transparent while maintaining an WCAG AA-compliant contrast ratio (tested
+  // using the TPGi Color Contrast Analyzer application) even on top of a solid
+  // white background.
+  @include linear-gradient(
+    180deg,
+    rgba(0, 0, 0, 0.9) 0%,
+    rgba(0, 0, 0, 0.7) 60%,
+    rgba(0, 0, 0, 0) 100%
+  );
+  font-size: 1.2em; // 12px
+  line-height: 1.5; // 18px
+  @include transition(opacity 0.1s);
+  padding: 0.666em 1.333em 4em; // 8px 16px 48px
+  pointer-events: none;
+  position: absolute;
+  top: 0;
+  width: 100%;
+}
+
+.vjs-title-bar-title,
+.vjs-title-bar-description {
+  margin: 0;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+
+.vjs-title-bar-title {
+  font-weight: bold;
+  margin-bottom: 0.333em; // 4px
+}
+
+.vjs-playing.vjs-user-inactive .vjs-title-bar {
+  opacity: 0;
```

Comment:
Not specific to just the title bar, but this is a good time to think about adding it, if on fullscreen we set `display: none` on anything we overlay and `background-color: transparent` on the fullscreen element then [Safari's low power mode](https://developer.apple.com/documentation/webkit/delivering_video_content_for_safari) can kick in. `opacity: 0` or `visibility: hidden` aren't invisible enough of course.

```css
.vjs-playing.vjs-fullscreen,
.vjs-playing.vjs-fullscreen .vjs-tech {
  background-color: transparent;
}
.vjs-playing.vjs-fullscreen.vjs-user-inactive .vjs-title-bar
.vjs-playing.vjs-fullscreen.vjs-user-inactive .vjs-control-bar {
  display: none;
}
```

Reply:
That's a good thing to discuss. That doc says that the background should be black, not transparent. And it says `Set the CSS property display:none for all non-video elements on your webpage while the video is playing`... does that mean _all_ elements or just those that are part of the fullscreen experience?

Either way, setting `display:none` on all elements except the `video` element seems like a non-starter because we lose all our UI and any other implementations like overlays or ads.

Reply:
Also, is this something that we can only do in a major version?

Reply:
[This blog](https://squaredi.blogspot.com/2021/05/) explains it better than Apple's documention, they're right that `background-color: transparent` rather than `black` is necessary for low power mode when teting with the quart debug tool.

`display: none` would only be while the user is inactive. if they tab, then they're not inactive and it would not longer be `display: none`

Reply:
The issue is that with a screen reader, you can still be interacting with a control when the controls visually hide. Since our compromise was hiding the controls even if it has focus but keeping it accessible for Screen Readers.
For regular keyboard control, I think it would've been fine, but we specifically want SRs to maintain access to the controls even if the controls are visually hidden.

Reply:
heh, yeah. It would be cool to investigate how if we can take advantage of low power mode. Maybe a carve out for fullscreen on safari? Not sure.